x-pack/test/detection_engine_api_integration/utils.ts

```diff
@@ -721,7 +721,7 @@ export const getSimpleRuleOutputWithWebHookAction = (actionId: string): Partial<
 export const waitFor = async (
   functionToTest: () => Promise<boolean>,
   functionName: string,
-  maxTimeout: number = 10000,
```

Comment:
Nit: Is there a way to modify this at the assertion level, rather than making this change globally?

Reply:
Yeah, you can bubble it up. Right now, it's defaulting so it's capable of bubbling up. I figured this was better to change globally if we saw one instance of it failing.